app/src/main/java/com/framgia/wsm/screen/requestleave/RequestLeavePresenter.java

```diff
@@ -6,13 +6,15 @@
 import com.framgia.wsm.data.source.UserRepository;
 import com.framgia.wsm.data.source.remote.api.error.BaseException;
 import com.framgia.wsm.data.source.remote.api.error.RequestError;
+import com.framgia.wsm.utils.Constant;
 import com.framgia.wsm.utils.common.StringUtils;
 import com.framgia.wsm.utils.rx.BaseSchedulerProvider;
 import com.framgia.wsm.utils.validator.Validator;
 import io.reactivex.annotations.NonNull;
 import io.reactivex.disposables.CompositeDisposable;
 import io.reactivex.disposables.Disposable;
 import io.reactivex.functions.Consumer;
+import retrofit2.http.HEAD;
```

Comment:
ERROR: Unused import: retrofit2.http.HEAD. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports